### PR TITLE
Just checking version import

### PR DESCRIPTION
**Pull request by IAlibay**
In this PR:
- Check version import in CI
- fixes universal wheel

### CODE REVIEW

**Comment by IAlibay** [comment]
@mikemhenry the version installs as `0.0.0` - this seems weird? (https://github.com/OpenFreeEnergy/gufe/actions/runs/4621160585/jobs/8172233914?pr=165)

**Comment by IAlibay** [comment]
nevermind, this is a checkout depth issue

File: pyproject.toml [hunk 1]
```diff
@@ -43,9 +43,6 @@ license-files = ["LICENSE"]
 [tool.setuptools.packages]
 find = {namespaces = false}
 
-[tool.distutils.bdist_wheel]
```
Comment (IAlibay) [inline pyproject.toml:46]: @mikemhenry - universal bdist_wheel entries are for pure Python projects that are both py2 and py3 compliant (you can see the effect when you build a wheel, it'll output "py2py3" in the name)

**Comment by mikemhenry** [comment]
And yes, I have it setup to use 0.0.0 if I can't find tag info instead of error out, so this behavior is expected. A depth of 0 should be fine since our repo is small

**Review by mikemhenry — APPROVED** [review]

**Comment by mikemhenry** [comment]
Also @IAlibay would you mind doing this for openfe as well?

**Comment by IAlibay** [comment]
yup will do

**Pull request merged**